jennchenn/uwexplore
# small style changes, can login w enter key

## Implementation description

- pressing enter in login fields will trigger submit event to log the user in
- more space between the add button, and collapse button
- changed text colour on CEAB term dropdown to be more readable
- centered search card content, and added padding to titles in the calendar tray
- hopefully put back in fix to remove extra padding on search cards -> as seen in demo 



## Steps to test

1. try logging in w a user and pressing "enter" in the password field -> should get logged in without having to click a button



## What should reviewers focus on?

-
